### PR TITLE
Adding Marten since he advises CS students, and Amir who moved

### PR DESCRIPTION
**Pull request by benjaminfuller**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by benjaminfuller** [comment]
Marten van Dijk has home department of ECE but joint with CSE and advises CSE PhD students.  Amir Herzberg has changed from Bar Ilan to UConn full time.

File: csrankings.csv [hunk 1]
```diff
@@ -7930,6 +7930,7 @@ Marta L. Queiros Mattoso,UFRJ,http://cos.ufrj.br/~marta,ZPczFBAAAAAJ
 Marta M. Betcke,University College London,http://www0.cs.ucl.ac.uk/people/M.Betcke.html,HJ3z-24AAAAJ
 Marta Mattoso,UFRJ,http://cos.ufrj.br/~marta,ZPczFBAAAAAJ
 Marta Mejail,University of Buenos Aires,https://www.dc.uba.ar/rrhh/profesores/mejail,WMy2FhwAAAAJ
+Marten van Dijk,University of Connecticut,https://scl.engr.uconn.edu/,byCWPiwAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:7933]: Not listed here: https://www.cse.uconn.edu/people/faculty/
Please read over inclusion criteria.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please remove the one faculty member from EE or provide evidence that he can solely advise PhD students _in CS_ (for example, an indication of an adjunct or affiliate appointment).

File: csrankings.csv (hunk 1 above)
Comment (benjaminfuller) [reply csrankings.csv:7933]: Including appointment letter from department.  Primary affiliation is ECE but can/does supervise PhD students in CSE.



martenappointment

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
That will do it. Thanks!

**Pull request merged**